unicoremc/views.py

```diff
@@ -75,6 +75,7 @@ def form_valid(self, form):
 
         project = self.get_object()
         project.create_pyramid_settings()
+        project.create_nginx()
```

Comment:
This usally happens in the workflow when a new project is created. We never needed to recreate nginx configs in the views before, only the pyramid settings.